clients/js/alph.ts

```diff
@@ -3,121 +3,115 @@ import { impossible } from './utils'
 import { web3, Script, SubmissionResult, subContractId } from '@alephium/web3'
 import { PrivateKeyWallet } from '@alephium/web3-wallet'
 import {
-  Contracts,
-  CONTRACTS,
-  updateGuardianSetScript,
-  upgradeGovernanceContractScript,
-  createRemoteTokenPoolOnAlph,
-  upgradeTokenBridgeContractScript,
-  setMessageFeeScript,
-  transferFeeScript,
-  destroyUnexecutedSequencesScript,
-  updateMinimalConsistencyLevelScript,
-  updateRefundAddressScript,
-  GovernancePayload
+    Contracts,
+    CONTRACTS,
+    updateGuardianSetScript,
+    upgradeGovernanceContractScript,
+    createRemoteTokenPoolOnAlph,
+    upgradeTokenBridgeContractScript,
+    setMessageFeeScript,
+    transferFeeScript,
+    destroyUnexecutedSequencesScript,
+    updateMinimalConsistencyLevelScript,
+    updateRefundAddressScript,
+    registerChain,
+    GovernancePayload
 } from 'alephium-wormhole-sdk'
 
 export async function executeGovernanceAlph(
-  payload: GovernancePayload,
-  vaa: Buffer,
-  network: 'MAINNET' | 'TESTNET' | 'DEVNET'
+    payload: GovernancePayload,
```

Comment:
We use 2 spaces in general in the codebase. Let's be consistent.
The diff looks confusing now.

Reply:
Fixed the [tsfmt.json](https://github.com/alephium/wormhole-fork/blob/explorer-and-cloudfunctions-testnet/clients/js/tsfmt.json#L2).